pkg/client/dtclient/config_client.go

```diff
@@ -35,7 +35,36 @@ import (
 	"strings"
 )
 
-func (d *DynatraceClient) upsertDynatraceObject(ctx context.Context, theApi api.API, objectName string, payload []byte) (DynatraceEntity, error) {
+// APIData holds information about the api of the config to be read or written on the server
+type APIData struct {
```

Comment:
I am open for better names ... just didn't want to name it API as well, to keep confusion at a minimum